memgpt/agent.py

```diff
@@ -445,7 +451,8 @@ def contains_special_characters(s):
             if contains_special_characters(monologue):
                 printd(f"First message internal monologue contained special characters: {response_message}")
                 return False
-            if 'functions' in monologue or 'send_message' in monologue or 'inner thought' in monologue.lower():
+            # if 'functions' in monologue or 'send_message' in monologue or 'inner thought' in monologue.lower():
+            if 'functions' in monologue or 'send_message' in monologue:

```

Comment:
I think it's fine to drop this check overall (not just for gpt-3.5). What do you think @vivi ?

Reply:
sgtm, if it starts impacting experience we can reinstate the check